app/src/lib/api.ts

```diff
@@ -57,6 +57,14 @@ export interface IAPICommit {
   readonly author: IAPIUser | null
 }
 
+export type Plan = 'developer' | 'free' | 'team' | 'business'
+
+interface IAccountWithPlan {
```

Comment:
Could you append API to this name to make it clear this interface should only be used when parsing API results?